src/renderer/rendererutils.ts

```diff
@@ -1,9 +1,5 @@
 const getVideoResult = (video: any): boolean => {
```

Comment:
You should remove it entirely at this point when it only returns whatever it's passed.

Reply:
hilarious - how did I not think of this